frontend/api/types/Fetch.ts

```diff
@@ -1,5 +1,8 @@
 export enum FetchUrl {
     LOGIN = "/backend/auth",
+    USERS = "/backend/users",
+    POST = "/api/posts",
```

Comment:
we should use /backend OR /api 
/api seems better and shorter but then update it everywhere